src/base/emu-i386/kvm.c

```diff
@@ -361,13 +361,23 @@ static int init_kvm_vcpu(void)
      this is only really needed if the vcpu is started in real mode and
      the kernel needs to emulate that using V86 mode, as is necessary
      on Nehalem and earlier Intel CPUs */
+  ret = ioctl(kvmfd, KVM_CHECK_EXTENSION, KVM_CAP_SET_TSS_ADDR);
```

Comment:
Ah, its the same func...
How about moving cap checks to
`kvm_init_cpu()`? I think they are
here by mistake, because failure
from `kvm_init_cpu()` results in a
selection of different back-end,
whereas here it would just terminate.

Reply:
ok, will do

Reply:
done in 6fb34723